web/libs/editor/src/tags/object/Video/Video.js

```diff
@@ -183,6 +187,23 @@ const Model = types
       if (!framerate || Number.isNaN(framerate)) self.framerate = "24";
       else if (framerate < 1) self.framerate = String(1 / framerate);
       else self.framerate = String(framerate);
+
+      // normalize playback speed parameters
+      const defaultPlaybackSpeed = Number(parseValue(self.defaultplaybackspeed, self.store.task?.dataObj));
+      const minPlaybackSpeed = Number(parseValue(self.minplaybackspeed, self.store.task?.dataObj));
+
+      // validate and set minPlaybackSpeed
+      self.minplaybackspeed =
+        !minPlaybackSpeed || Number.isNaN(minPlaybackSpeed) || minPlaybackSpeed < 0.25 ? 1 : minPlaybackSpeed;
+
+      // validate and set defaultPlaybackSpeed
+      self.defaultplaybackspeed =
+        !defaultPlaybackSpeed || Number.isNaN(defaultPlaybackSpeed) || defaultPlaybackSpeed < 0.25
+          ? 1
+          : Math.max(defaultPlaybackSpeed, self.minplaybackspeed);
```

Comment:
We'll leave default minimum value as it is right now to not break existing experience: 0.25.
But the absolute minimum for both values should be the closest value to zero given our step in UI, which is 0.05.

Also `Number.isNaN()` is a very simple check, we need usual `isNaN()` for better check with type coercion.

```suggestion
      self.minplaybackspeed =
        !minPlaybackSpeed || isNaN(minPlaybackSpeed) || minPlaybackSpeed < 0.05 ? 0.25 : minPlaybackSpeed;

      // validate and set defaultPlaybackSpeed
      self.defaultplaybackspeed =
        !defaultPlaybackSpeed || isNaN(defaultPlaybackSpeed) || defaultPlaybackSpeed < 0.05
          ? 1
          : Math.max(defaultPlaybackSpeed, self.minplaybackspeed);
```